docassemble/ALToolbox/data/static/review_widget.js

```diff
@@ -0,0 +1,56 @@
+var al_fade_speed = 300;
+
+function after_click() {
+  var thumbs_stage = document.getElementsByClassName('al-thumbs-widget');
+  var feedback_stage = document.getElementsByClassName('al-review-text');
+
+  for (let elem of thumbs_stage) {
```

Comment:
What versions of ie are we supporting? This uses `let`, which is generally good practice for `for`s, but is only supported from ie 11 onwards. If that's where we're at, the practice is generally used everywhere in the code.

Just as a point of information, I believe `var` wouldn't run into any problems for the code inside the `for` here.

Reply:
Very good point, probably should have used `var`. TBF I'm not sure if everything else we do / docassemble / bootstrap does supports IE < 11, but it would be silly if everything else is fine but not this.